frontend-web/webclient/app/UserSettings/ChangeNotificationSettings.tsx

```diff
@@ -0,0 +1,128 @@
+import {useCloudCommand} from "@/Authentication/DataHook";
+import * as React from "react";
+import {useCallback, useEffect} from "react";
+import {Box, Button, Checkbox, Label} from "@/ui-components";
+import * as Heading from "@/ui-components/Heading";
+import {snackbarStore} from "@/Snackbar/SnackbarStore";
+import {bulkRequestOf} from "@/DefaultObjects";
+import HexSpin from "@/LoadingIcon/LoadingIcon";
+import {NotificationSettings, retrieveNotificationSettings, updateNotificationSettings} from ".";
+
+interface UserDetailsState {
+    settings: NotificationSettings
+}
+
+export const defaultNotificationSettings: NotificationSettings = {
+    jobStarted: true,
+    jobStopped: true,
+}
+
+export enum NotificationType {
+    JOB_STARTED,
+    JOB_STOPPED,
+}
+
+const initialState: UserDetailsState = {
+    settings: defaultNotificationSettings
+};
+
+type UpdatePlaceholdersNotificationSettings = PayloadAction<"UpdatePlaceholdersNotificationSettings", UserDetailsState>;
+
+const reducer = (state: UserDetailsState, action: UpdatePlaceholdersNotificationSettings): UserDetailsState => {
+    switch (action.type) {
+        case "UpdatePlaceholdersNotificationSettings":
+            return {...state, ...action.payload};
+    }
+};
+
+export const ChangeNotificationSettings: React.FunctionComponent<{setLoading: (loading: boolean) => void}> = () => {
+    const [commandLoading, invokeCommand] = useCloudCommand();
+    const [state, dispatch] = React.useReducer(reducer, initialState, () => initialState);
+    const info = useCallback(async () => {
+
+        const notificationSettings = await invokeCommand(
+            retrieveNotificationSettings({}),
+            { defaultErrorHandler: false }
+        );
```

Comment:
If you're using `defaultErrorHandler: false`, shouldn't you wrap this in try-catch?